schema/pnts.featureTable.schema.json

```diff
@@ -40,6 +43,9 @@
             },
             "CONSTANT_RGBA" : {
                 "$ref" : "featureTable.schema.json#/definitions/globalPropertyCartesian3"
+            },
+            "BATCH_LENGTH" : {
```

Comment:
It's needed because we expect multiple points to map to the same batchId. In the case of per-point properties, both `BATCH_ID` and `BATCH_LENGTH` can be omitted.